packages/plans-grid-next/src/hooks/data-store/use-grid-plan-for-spotlight.ts

```diff
@@ -0,0 +1,31 @@
+import { getPlanClass } from '@automattic/calypso-products';
+import { useMemo } from 'react';
+import { GridPlan } from '../../types';
+
+const SPOTLIGHT_ENABLED_INTENTS = [ 'plans-default-wpcom' ];
+
+interface Params {
+	gridPlansForFeaturesGrid: GridPlan[];
+	intent: string | undefined;
```

Comment:
Should this be:

```suggestion
	intent?: PlansIntent;
```

Although, also thinking whether `intent` should ever be optional/undefined. It feels like none of these data hooks would produce anything meaningful without an intent (in which case it would make it a required prop) 🤔 

Sorry if I'm missing something. Let me know what you think

Reply:
Let's definitely address/discuss this before shipping. It's important that we get the props correctly :-)

Reply:
👍  Updated with the correct `PlansIntent` type: [e50a840](https://github.com/Automattic/wp-calypso/pull/87377/commits/e50a840f8ea4997ba04b1a3b8f4c9d33e3af6bfa)